File: FrontEnd/frontend-app/src/app/Model/audit-log-admin-response.ts
export interface AuditLogAdminResponse {
  id: number;
  productId: number;
  sku: string;
  productName: string;
  type: string;
  quantityChanged: number;
  previousQuantity: number;
  newQuantity: number;
  unitCostSnapshot: number;
  unitPriceSnapshot: number;
  performedBy: string;
  createdAt: string;
}

File: BackEnd/Inventory Management System/src/main/java/com/example/inventorymanagementsystem/DTO/AuditLogAdminResponse.java
package com.example.inventorymanagementsystem.DTO;

import java.math.BigDecimal;
import java.time.Instant;

public record AuditLogAdminResponse(
        Long id,
        Long productId,
        String sku,
        String productName,
        String type,
        Integer quantityChanged,
        Integer previousQuantity,
        Integer newQuantity,
        BigDecimal unitCostSnapshot,
        BigDecimal unitPriceSnapshot,
        String performedBy,
        Instant createdAt
) {
}
